vimeo-networking/src/main/java/com/vimeo/networking/VimeoService.java

```diff
@@ -357,6 +355,9 @@
 
     @GET
     Call<TextTrackList> getTextTrackList(@Header("Authorization") String authHeader, @Url String uri);
+
```

Comment:
Don't you still need to declare an endpoint for getting a product object?

Reply:
Not sure what we usually do here...

Above you said:
> If the new endpoints aren't top level, but are available off of the metadata of the User object, you should take the uri as a string parameter rather than hard-coding the endpoint.

Since `Product` contains `uri`field to get product details should we expect clients to use that, or provide a hardcoded endpoint?

Reply:
Follow up happened IRL.